src/helpers/Workflows/Workflows.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Layouts.
+ * @module constants/Workflows
+ */
+
+import config from '@plone/volto/registry';
+import { last, split } from 'lodash';
+
+/**
+ * Maps the available transitions to the current list of colors and returns a
+ * react-select option object
+ * @param  {Object} transition
+ *  {id: "published", title: "Published"}
+ * @return {Object} The react-select options object
+ *  { value: 'published', label: 'Published', color: '#007bc1' }
+ */
+export function getCurrentStateMapping(state) {
+  const mapping = config.settings.workflowMapping;
+
+  if (state.id in mapping) {
+    return { label: state.title, ...mapping[state.id] };
+  }
+
+  // Return an option with a neutral color
+  return {
+    value: state,
+    label: state,
+    color: '#000',
+  };
+}
+
+/**
+ * Maps the available transitions to the current list of colors and returns a
+ * react-select option object
+ * @param  {Object} transition
+ *  {
+ *     '@id': 'http://localhost:3000/de/@workflow/publish',
+ *     title: 'Veröffentlichen'
+ *  }
+ * @return {Object} The react-select options object
+ *  { value: 'public', label: 'Published', color: '#007bc1', url: 'http://localhost:3000/de/@workflow/publish' }
+ */
+export function getWorkflowOptions(transition) {
+  const mapping = config.settings.workflowMapping;
```

Comment:
this should be documented here https://6.dev-docs.plone.org/volto/configuration/settings-reference.html